### PR TITLE
doc: Improve readability, remove migration stuff

### LINKED ISSUE #488: Clean up notes, fix links
#### Summary

1. A lot of the notes and inlines are unnecessary and give the impression that the reader can skip through reading only the highlighted parts. Let's remove the notes, everything important should be in the main text, and notes can be used to summarize or provide additional info. In many docs we use {note} next to {warning] next to > and the whole doc is highlighted. Let's remove as much of that as possible.

2. Links should be [Text](Link), not using html. In content/the-things-stack there are already many <a href> which should not be there.

Bildschirmfoto 2021-11-18 um 10 48 20

#### Why do we need this ?

Improve the flow and visual appearance of docs, and make the markdown easier to read.

### PR DESCRIPTION
**Pull request by nejraselimovic**
#### Summary
Closes #488 

#### Notes for Reviewers
You'll notice I deleted the whole `/content/the-things-stack` folder. I think we should completely remove it because December 1 is almost here and I assumed this PR won't get merged before that. The only useful info was about TTS features, so I kept and incorporated that as part of the Quick Start section. Everything else was about migration and will no longer be needed in a few days.

Taking this into account, I think there's no too many notes or bad links.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links.
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

**Comment by jpmeijers** [comment]
The docs on using the migration tool is still useful for migrating from private V2 instances, Chirpstack instance, and usage of the migration tool in general. I refer back to this quite often for reasons other than migration. Recently I used it to move an end device from one application to another on the V3 stack.

**Comment by nejraselimovic** [comment]
> The docs on using the migration tool is still useful for migrating from private V2 instances, Chirpstack instance, and usage of the migration tool in general. I refer back to this quite often for reasons other than migration. Recently I used it to move an end device from one application to another on the V3 stack.

We will keep all that documentation on https://www.thethingsindustries.com/docs/.

Some time ago we decided that ttn.org documentation should contain exclusively what's relevant for TTS Community Edition and TTN V2 public community network, so I think documentation about migration which will be applicable for private instances doesn't belong here. People who use private instances should read tti.com docs and follow instructions there. And for migrating devices from one app to another on the same instance - I don't recall that we mention that use case anywhere in our documentation (although we could), it's kind of a special case. We could add a link to tti.com migration tool docs and say "if you need to migrate devices from one app to another, check out this tool", but I don't think we need to have a whole section on ttn.org dedicated to migration just for that. 

The migration content on this page was supposed to be just a shorter and simplified version on what we have on tti.com migration docs, so that community users could easily migrate from TTN V2 PCN to TTS Community Edition. For all other cases they should refer to tti.com migration docs. 

@benolayinka please let us know what you think

**Comment by jpmeijers** [comment]
Yes, the tti.com docs address everything that I need, so I agree with you.

File: doc/content/devices-and-gateways/adding-gateways/_index.md [hunk 1]
```diff
@@ -3,12 +3,14 @@ title: Adding Gateways
 section: Gateways
 ---
 
-To connect your gateway to The Things Stack Community Edition, it has to be configured to properly forward the traffic to it, and it also has to be added/registered on The Things Stack Community Edition. Instructions for adding gateways are available at [The Things Stack - Gateways](https://www.thethingsindustries.com/docs/gateways/).
+To connect your gateway to The Things Stack Community Edition, it has to be configured to properly forward the traffic to it, and it also has to be added/registered on The Things Stack Community Edition.
 
-It is recommended to use [LoRa Basics Station protocol](https://www.thethingsindustries.com/docs/gateways/lora-basics-station/), due to a number of security and scalability issues related to the [UDP Packet Forwarder](https://www.thethingsindustries.com/docs/gateways/semtech-udp-packet-forwarder/).
+{{< note >}} It is recommended to use [LoRa Basics Station protocol](https://www.thethingsindustries.com/docs/gateways/lora-basics-station/), due to a number of security and scalability issues related to the [UDP Packet Forwarder](https://www.thethingsindustries.com/docs/gateways/semtech-udp-packet-forwarder/). {{</ note >}}
```
Comment (benolayinka) [inline doc/content/devices-and-gateways/adding-gateways/_index.md]: dont think this needs to be a note

File: doc/content/devices-and-gateways/adding-devices/_index.md [hunk 2]
```diff
@@ -23,8 +23,8 @@ You can also see [examples](https://github.com/TheThingsNetwork/lorawan-devices/
 
 ## Adding a device using CLI
 
-To add a device using CLI, you need to have the CLI installed on your system. [Learn how to install and login with the CLI](https://www.thethingsindustries.com/docs/getting-started/cli/).
+{{< note >}} To add a device using CLI, you need to have the CLI installed on your system. [Learn how to install and login with the CLI](https://www.thethingsindustries.com/docs/getting-started/cli/). {{</ note >}}
```
Comment (benolayinka) [inline doc/content/devices-and-gateways/adding-devices/_index.md]: Not a note

File: doc/content/devices-and-gateways/bytes/_index.md [hunk 3]
```diff
@@ -160,7 +168,7 @@ int myVal = ((int)(payload[0]) << 8)
 
 If the range of expected values is bigger than 65536 we can use the same trick. The only difference is that we have to manually shift bits when we encode on Arduino, just like we did in the payload function.
 
-Let's say we need to encode a [long](https://www.arduino.cc/en/Reference/Long) which uses 4 bytes for a range up to 4294967296.
+Let us say we need to encode a [long](https://www.arduino.cc/en/Reference/Long) which uses 4 bytes for a range up to 4294967296.
```
Comment (benolayinka) [inline doc/content/devices-and-gateways/bytes/_index.md]: Thanks for correcting these, but I don't think we need to keep nitpicking the language this way. Nobody would ever say "Let us" and I disagree in general that the contractions are a problem.

File: doc/content/devices-and-gateways/bytes/_index.md [hunk 4]
```diff
@@ -280,7 +288,7 @@ memcpy(payloadA,
 
 The short answer is: **don't**. Text uses a lot of bytes. [Unicode](https://en.wikipedia.org/wiki/Unicode) defines more than 128000 characters, so that would take 3 bytes per character! There are rarely good reasons to use text instead of numbers, apart from maybe transmitting some user input. Most of the time only the Alpha-numeric characters suffice, in that case you can get away by using [ASCII characters](https://en.wikipedia.org/wiki/ASCII#Printable_characters) that only use a single byte per character. Every string must be terminated with a NULL (0x00, '\0') character to indicate the string has ended.
 
-You didn't hear it from me, but here's how you'd encode a string:
+You did not hear it from me, but here is how you would encode a string:
```
Comment (benolayinka) [inline doc/content/devices-and-gateways/bytes/_index.md]: Sounds so weird to me as a native speaker 😀 like a robot talking

**Review by benolayinka — APPROVED** [review]

**Pull request merged**